gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -6660,13 +6702,17 @@ subroutine interstitial_setup_tracers(Interstitial, Model)
       do n=2,Model%ntrac
         if ( n /= Model%ntcw  .and. n /= Model%ntiw  .and. n /= Model%ntclamt .and. &
              n /= Model%ntrw  .and. n /= Model%ntsw  .and. n /= Model%ntrnc   .and. &
-             n /= Model%ntsnc .and. n /= Model%ntgl  .and. n /= Model%ntgnc) then
+!            n /= Model%ntlnc .and. n /= Model%ntinc .and.                          &
+             n /= Model%ntsnc .and. n /= Model%ntgl  .and. n /= Model%ntgnc   .and. &
+             n /= Model%nthl  .and. n /= Model%nthnc .and.                          &
+             n /= Model%nqvolg .and. n /= Model%nqvolh ) then
           tracers = tracers + 1
           if (Model%ntke  == n ) then
             Interstitial%otspt(tracers+1,1) = .false.
             Interstitial%ntk = tracers
           endif
-          if (Model%ntlnc == n .or. Model%ntinc == n .or. Model%ntrnc == n .or. Model%ntsnc == n .or. Model%ntgnc == n)    &
+          if (Model%ntlnc == n .or. Model%ntinc == n .or. Model%ntrnc == n .or. Model%ntsnc == n .or. &
+               Model%ntgnc == n .or. Model%nthnc == n .or. Model%nqvolg == n .or. Model%nqvolh == n)    &
```

Comment:
@grantfirl @tsupinie 
I have been working a parallel effort on this and did a similar thing here (in the SCM first). It seems that this if statement doesn't matter for ntrnc and others that are excluded by the outer if conditional (i.e., n /= model%ntrnc). It looks like only ntlnc == n and ntinc == n would ever trigger this, right?

Reply:
Yeah, I think that's correct. It looks like that controls which tracers are advected in updrafts and downdrafts in the Chikira-Sugiyama convective scheme, which I've never used. At some point, we'll probably want to run tests to see what settings are best. For now, it's a bit over-complicated, I guess.

Reply:
To be honest I haven't really figured out the subgrid (shallow/deep) convective transport yet. The GFS_suite_interstitial_3_run and GFS_suite_interstitial_4_run routines have a duplicate logic that presumably could be replaced by the otspt array? I'm just looking at 3km dx, so I have those modules removed from the working suite, anyway.

Reply:
@MicroTed, do you have a branch where you have the SCM changes for this scheme? If so, could you point me to it? I'm generally more comfortable testing new schemes out there before FV3, so that would help me out for reviewing this again. Hopefully, we can get this regression tested with the FV3 and get this merged in to master this time around. :)